packages/dart_frog/test/src/serve_test.dart

```diff
@@ -25,5 +25,22 @@ void main() {
       expect(response.statusCode, equals(HttpStatus.notFound));
       await server.close();
     });
+
+    test('exposes connectionInfo on the incoming request', () async {
+      late HttpConnectionInfo connectionInfo;
+      final server = await serve(
+        (context) {
+          connectionInfo = context.request.connectionInfo;
+          return Response();
+        },
+        'localhost',
+        3000,
+      );
+      final client = HttpClient();
+      final request = await client.getUrl(Uri.parse('http://localhost:3000'));
+      await request.close();
+      expect(connectionInfo, isA<HttpConnectionInfo>());
```

Comment:
Should we also test for the values? 

This technically is guaranteed to be this type otherwise the test would have failed beforehand.

Reply:
Won’t the values be inconsistent?

Reply:
Updated to verify the address